.github/workflows/verify-images.yaml

```diff
@@ -0,0 +1,25 @@
+name: Verify Docker Compose Images
+
+on:
+  push:
+    paths:
+      - '**.yaml'
```

Comment:
I'll merge, but could this catch too many files?

Reply:
There are four files with extension `.yaml`. Rest three are in `.github/workflows/`

Reply:
This matches any YAML file in the project, don't we just need this to trigger on changes to: `/docker-compose.yaml`?

Reply:
I had given it try with many formats, before raising this PR. It doesn't work

1. `/docker-compose.yaml`
2. `./docker-compose.yaml`
3. `docker-compose.yaml`